rna/models.py

```diff
@@ -44,6 +56,7 @@ class Release(TimeStampedModel):
     bug_list = models.TextField(blank=True)
     bug_search_url = models.CharField(max_length=2000, blank=True)
     system_requirements = models.TextField(blank=True)
+    system_requirements_id = models.ForeignKey(SystemRequirement, blank=False, on_delete=models.CASCADE)
```

Comment:
I don't think this will work. Django likes to add `_id` to the actual database column names for foreign keys, so I guess it would be `system_requirements_id_id`? Might work. What I was thinking was more along the lines of changing the existing column name to something like `system_requirements_custom` and name the new one `_system_requirements`, and add a property method to the class:

```python
@property
def system_requirements(self):
    return self.system_requirements_custom or self._system_requirements.content
```